packages/feedback-service/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:alpine
+FROM node:12-alpine
```

Comment:
Can we use 14 LTS version

Reply:
The microservices are built with 12, so we'll have to first make sure all the microservices work inn node:14.

After that, it's just about bumping the number from 12 to 14.